site/doc/1.11/journal.md

```diff
@@ -356,6 +356,38 @@ when an amountless posting is balanced using a price's commodity, or
 when -V is used.) If you find this causing problems, set the desired
 format with a commodity directive.
 
+#### Amount Expressions
+
+An amount may also be comprised of multiple parts joined by basic
+arithmetic operators; in this case, each individual value follows the
+structure described above, while the joins allow:
+
+-   addition, subtraction, and multiplication using the standard ASCII
+    operators (`+`, `-`, `*`), either adjacent to the amount(s) or
+    separated by spaces
+-   parenthesised (sub-)expressions, to be evaluated before any
+    operations outside them. Note, however, that all commodity symbols
+    must be attached to an amount inside the parentheses: `($10 * 0.2)`
+    works, `$(10 * 0.2)` does not.
+-   multiple commodities in the same expression
+
+Multiplication of commodities is handled a bit differently than may be
+expected; if the amount to the right of the operator includes an
+explicit commodity, that commodity replaces any to the left for
+compatibility with [automated postings](#automated-postings). If that
+right value includes multiple commodities itself (parenthesised to obey
+the order of operations), the left value is multiplied by each
+individually.
+
+As examples, with the value each expression simplifies to (if
+applicable):
+
+`$47.18-$7.13` -> `$40.05`
+`$47.18 - ($20 + $7.13)` -> `$20.05`
+`$6.10 + £4.35 * 0.8 €` -> `$6.10 + 3.48 €`
+`($6.10 + £4.35) * 0.8 €` -> `8.36 €`
+`$6.10 * (£0.2 + 0.8 €)` -> `£1.22 + 4.88 €`
+
```

Comment:
I think folks will immediately want division (`/`) as well, can you add that ? We'll need to handle division by zero in some reasonable manner.

Reply:
Adding it won't be hard at all, except for divide-by-zero; the operation parser wound up being amazingly extensible.  I'm tempted to just make it silently fail, but it's probably better to work through the error reporting system to ensure sanity.  I've not looked at that yet, though, so I might put division off until the end of the refactor, if you think that's all right.

Reply:
division can be a problem too... What would be the amount for `$0.05 / 3` ?

Reply:
1.6666666666666666...

Reply:
I meant: 0.016666666666666666...